go/integTest/shared_commands_test.go

```diff
@@ -1040,6 +1040,39 @@ func (suite *GlideTestSuite) TestHStrLen_WithNotExistingField() {
 	})
 }
 
+func (suite *GlideTestSuite) TestHIncrBy_WithExistingField() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := uuid.NewString()
+		field := uuid.NewString()
+		fieldValueMap := map[string]string{field: "10"}
+
+		hsetResult, err := client.HSet(key, fieldValueMap)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(1), hsetResult.Value())
+
+		hincrByResult, hincrByErr := client.HIncrBy(key, field, 1)
```

Comment:
We probably don't need a new err var here since the previous one is never used again, no?

Reply:
Not sure, about this change. I think both are the same.
hincrByErr, gives more context in the assert statement about which err is being checked.

Reply:
Since we are just checking the err value to be null to ensure the successful execution of the command I think the err variable is usable, but I guess I do see your point about readability. Resolving this since I'm fine with it.